### PR TITLE
406 407 command attr player level

### PR DESCRIPTION
**Pull request by MarcusSilvaDev**
image

### CODE REVIEW

File: data/extensions/Spells/Commands/AttrPlayerCommand.cs
Comment (caioavidal) [inline data/extensions/Spells/Commands/AttrPlayerCommand.cs:1]: can you avoid if/else?

**Review by caioavidal — CHANGES_REQUESTED** [review]
Can you fix the code smells and code factor ?

File: data/extensions/Spells/Commands/AttrPlayerCommand.cs
Comment (MarcusSilvaDev) [reply data/extensions/Spells/Commands/AttrPlayerCommand.cs:1]: I don`t want

Comment (MarcusSilvaDev) [reply data/extensions/Spells/Commands/AttrPlayerCommand.cs:1]: hahaha, i'll see brow

**Pull request merged**